storage/store.go

```diff
@@ -1324,8 +1319,7 @@ func (s *Store) ProposeRaftCommand(idKey cmdIDKey, cmd proto.InternalRaftCommand
 		panic("proposed a nil command")
 	}
 	// Lazily create group. TODO(bdarnell): make this non-lazy
-	err := s.multiraft.CreateGroup(uint64(cmd.RaftID))
-	if err != nil {
+	if err := s.multiraft.CreateGroup(uint64(cmd.RaftID)); err != nil {
```

Comment:
I thought we had a rule about not changing unrelated code for personal preference. I don't want to speak for Ben but I think I remember he preferred the original form in cases like these.